src/app/(guest)/exams/[id]/answer/page.tsx

```diff
@@ -175,24 +173,25 @@ const Exam: React.FC<ExamPageProps> = ({ params }) => {
 
           {currentQuestion ? (
             <section className="mb-10">
-              <p className="text-lg font-bold mt-5">{currentQuestion.question}</p>
-              <p className="text-sm text-gray-600 mt-2">
-                Tipo de pergunta '{currentQuestion.question_type}' do exame '{currentQuestion.exam}'
-              </p>
-              <div className="mt-5 space-y-5">
-                {currentQuestion.options.map((option) => (
-                  <div
-                    key={option.name}
-                    onClick={() => selectAnswer(currentQuestionIndex, option.order)}
-                    className={`w-full flex items-center px-5 py-3 border border-gray-100 h-20 rounded hover:cursor-pointer hover:bg-primary hover:text-white transition ease-in-out ${
-                      answers.get(currentQuestionIndex) === option.order && 'bg-primary text-white'
-                    }`}>
-                    <p>{option.name}</p>
-                    {answers.get(currentQuestionIndex) === option.order && (
-                      <Check className="ml-5" />
-                    )}
-                  </div>
-                ))}
+              <QuestionPrompt
+                currentQuestion={currentQuestion}
+                selectAnswer={selectAnswer}
+                currentQuestionIndex={currentQuestionIndex}
+                answers={answers}
+              />
+              <div className="w-full flex justify-center mt-10">
+                <PrimaryButton
```

Comment:
Rethink the design, probably two arrow icons on the side of the question numbers would better suit the design.
If you can, make some variations of the approach and annex them to the PR to discuss in the next meeting